index.html

```diff
@@ -245,12 +311,17 @@ <h1 class="display-4">An event-driven networking engine written in Python </h1>
                   <a class="ext-link" href="https://support.google.com/accounts/answer/185833?hl=en">generate one for gmail</a>,
                   <a class="ext-link" href="https://www.fastmail.com/help/clients/apppassword.html">generate one for fastmail</a>), and
                   <a class="ext-link" href="https://twistedmatrix.com/documents/current/core/howto/endpoints.html#auto7">
```

Comment:
any `https://twistedmatrix.com/documents/current` link should now be `https://docs.twistedmatrix.com/en/stable/`